docs/zkapps/o1js/index.mdx

```diff
@@ -60,7 +60,4 @@ Start your o1js journey by learning about [basic zk programming concepts](/zkapp
 
 Please see our page on [Security and zkApps](/zkapps/writing-a-zkapp/introduction-to-zkapps/secure-zkapps) for more information on ensuring your zkApp is secure.
 
-<<<<<<< HEAD:docs/zkapps/o1js/index.md
 Start your o1js journey by learning about [basic zk programming concepts](./o1js/basic-concepts).
-=======
->>>>>>> main:docs/zkapps/o1js/index.mdx
```

Comment:
this conflict was fixed incorrectly, it reintroduces a sentence that is out of context in this location

Reply:
fixed